compiler/src/main/kotlin/visitors/sanitychecker.kt

```diff
@@ -31,6 +31,29 @@ class SanityChecker : BaseASTVisitor() {
         if (worldHasEdge && node.edge == null) {
             ErrorLogger += SanityError(node.ctx, "A dimension is an [edge], but the edge state was not declared.")
         }
+
+        if (node.cellSize <= 0) {
+            ErrorLogger += SanityError(
+                node.ctx, "Cellsize must be an integer larger than 0, but was found to be" +
```

Comment:
```suggestion
                node.ctx, "Cellsize must be an integer larger than 0, but was: " +
```

Setting the value after a colon indicates that it is an internal value moreso than something written directly into error-template. 

This applies for the following two.